docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to the Rust Exercism Track
+
+This document is a work-in-progress.
+Current maintainers are primarily preparing for the v3 launch of the Rust track.
+Nonetheless, feel free to peruse what we have written thus far!
+
+*Contributions welcome :)*
+
+Thank you so much for contributing! :tada:
```

Comment:
Though this line was inherited from previous documentation, I feel that it fits poorly here. Probably the best options might be to move it back to the first non-header line of the document, or remove it.

Reply:
Great point. I feel removing is best!